src/Tests/FakeServiceBusClient.cs

```diff
@@ -25,5 +26,15 @@ public override ServiceBusSender CreateSender(string queueOrTopicName, ServiceBu
             }
             return Senders[queueOrTopicName];
         }
+
+        public override ServiceBusProcessor CreateProcessor(string queueName, ServiceBusProcessorOptions options)
+        {
+            if (!Processors.ContainsKey(queueName))
+            {
+                var fakeSender = new FakeProcessor();
+                Processors[queueName] = fakeSender;
+            }
+            return Processors[queueName];
+        }
```

Comment:
I've heard you like performance tips so I'll provide an alternative for your consideration :smile: (not that it will matter in this case)

```
            if (!Processors.TryGetValue(queueName, out var fakeSender))
            {
                fakeSender = new FakeProcessor();
                Processors.Add(queueName, fakeSender);
            }
            return fakeSender;
```

Reply:
Adressed :dancing_men: